us.cityofnewyork.data/ems-incident-dispatch/ems-incident-dispatch.yaml

```diff
@@ -11,39 +11,75 @@ content:
-          - first_to_hosp_datetime TIMESTAMP  # The date and time the first unit gives the signal that it is enroute to the hospital.
-          - first_hosp_arrival_datetime TIMESTAMP  # The date and time the first unit signals that it has arrived at the hospital.
-          - incident_close_datetime TIMESTAMP  # The date and time the incident closes in the dispatch system.
+          - incident_response_seconds_qy BIGINT  # The time elapsed in seconds between the incident_datetime and the first_on_scene_datetime.
+          - incident_travel_tm_seconds_qy BIGINT  # The time elapsed in seconds between the first_assignment_datetime and the first_on_scene_datetime.
+          - first_to_hosp_datetime STRING # The date and time the first unit gives the signal that it is enroute to the hospital.
+          - first_hosp_arrival_datetime STRING # The date and time the first unit signals that it has arrived at the hospital.
+          - incident_close_datetime STRING # The date and time the incident closes in the dispatch system.
           - held_indicator STRING  # Indicates that for some reason a unit could not be assigned immediately
-          - incident_disposition_code LONG  # A code indicating the final outcome of the incident. See incident dispositions.
+          - incident_disposition_code STRING # A code indicating the final outcome of the incident. See incident dispositions.
           - borough STRING  # The borough of the incident location.
           - incident_dispatch_area STRING  # The dispatch area of the incident.
           - zipcode STRING  # The zip code of the incident.
-          - policeprecinct LONG  # The police precinct of the incident.
-          - citycouncildistrict LONG  # The city council district.
-          - communitydistrict LONG  # The community district.
-          - communityschooldistrict LONG  # The community school district.
-          - congressionaldistrict LONG  # The congressional district.
+          - policeprecinct BIGINT  # The police precinct of the incident.
+          - citycouncildistrict BIGINT  # The city council district.
+          - communitydistrict BIGINT  # The community district.
+          - communityschooldistrict BIGINT  # The community school district.
+          - congressionaldistrict BIGINT  # The congressional district.
           - reopen_indicator STRING  # Indicates that at some point the incident was closed but then reopened.
           - special_event_indicator STRING  # Indicates that the incident was a special event such as the NYC Marathon.
           - standby_indicator STRING  # Indicates that the units were assigned to stand by in case they were needed.
           - transfer_indicator STRING  # Indicates that the incident was created for the transportation of a patient from one facility (ie a hospital or nursing home) to another.
+      preprocess:
```

Comment:
Can you please add a `# TODO: ` comment above pointing at the issue to support custom `timestampFormat` in DataFusion ingest?